Formula/vsh.rb

```diff
@@ -0,0 +1,18 @@
+class Vsh < Formula
+  desc "HashiCorp Vault interactive shell"
+  homepage "https://github.com/fishi0x01/vsh"
+  url "https://github.com/fishi0x01/vsh/archive/v0.7.2.tar.gz"
+  sha256 "f20885425a9c6ee04ed447046e6df113585469b37332e02b7a23639b718314a1"
+  license "MIT"
+
+  depends_on "go" => [:build]
```

Comment:
```suggestion
  depends_on "go" => :build
```
Sorry, missed this one.

Reply:
Good catch! Fixed.